src/openrct2-ui/input/KeyboardShortcut.cpp

```diff
@@ -98,7 +98,7 @@ static void ShortcutCloseAllFloatingWindow()
 {
     if (!(gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR))
         window_close_all();
-    else if (gS6Info.editor_step == EDITOR_STEP_LANDSCAPE_EDITOR)
+    else if (gS6Info.editor_step == RCT2EditorStep::LandscapeEditor)
```

Comment:
Why call this `RCT2EditorStep`? We only put this prefix if we use a different approach than RCT2.

Reply:
I just reused what was already there. What do you suggest, just `EditorStep`?

Reply:
Yup, just `EditorStep`.